content/blog/2024/10/03/2024-10-03-jenkins-election-candidates.adoc

```diff
@@ -0,0 +1,223 @@
+
+Affiliations: Schaeffler
+
+Profile Links:
+link:https://github.com/StefanSpieker[GitHub],
+link:https://www.linkedin.com/in/stefan-spieker-446168161/[LinkedIn],
+link:https://contributors.jenkins.io/pages/contributors/stefan-spieker/[Contributor spotlight],
+link:https://community.jenkins.io/u/stefanspieker/summary[Jenkins community],
+link:https://stackoverflow.com/users/1287864/s-spieker[Stackoverflow]
+
+[[jonesbusy]]
+=== link:/blog/authors/jonesbusy/[Valentin Delaye]
+
+I've been a Jenkins user since it's inception in 2011 and contributor since 2018.
+As of today I maintain more than 35 plugins.
+
+Starting of this year I got even more involved in the Jenkins community, by participating as a mentor for the Google Summer of Code to work on the Jenkins Plugin Modernizer Tool which is a CLI to update plugins at scale using OpenRewrite.
+
+I'm particularly interested in the "cloud ready" aspect of Jenkins including deployments, packaging (like Jenkinsfile Runner) or distributed storage.
+A recent contribution on this topic is the link:https://plugins.jenkins.io/artifactory-artifact-manager/[Artifactory Artifact Manager plugin] that I authored and maintain.
+
+If I'm elected on the Jenkins governance board those will be my subjects of choice.
+
+Affiliations: ELCA Cloud Services
+
+Profile Links:
+link:https://github.com/jonesbusy[GitHub],
+link:https://www.linkedin.com/in/valentindelaye/[LinkedIn],
+link:https://contributors.jenkins.io/pages/contributors/valentin-delaye/[Contributor spotlight],
+link:https://community.jenkins.io/u/jonesbusy/summary[Jenkins community]
+
+[[krisstern]]
+=== link:/blog/authors/krisstern/[Kris Stern]
+
+I have been a Jenkins contributor since 2021.
+Currently I am a maintainer of four plugins, have been a GSoC org admin and mentor for Jenkins for three years, as well as am a Jenkins Release Team member.
+Academically I have been trained as an observational astrophysicist and obtained my PhD degree from the University of Hong Kong (HKU) in 2021.
+I have worked professionally as a software engineer since late 2019, currently working in a AiFi startup.
+Currently I am also a part-time MCIT Online student at UPenn.
+I am passionate about open-source and would like to share my love of open-source software with the others.
+I feel grateful to be nominated. If elected, I would love to bring with me a startup mindset to help keep Jenkins perpetually youthful and relevant as well as a welcoming community to newcomers.
+
+Affiliations: GAIB.ai
+
+Profile Links:
+link:https://github.com/krisstern[GitHub],
+link:https://www.linkedin.com/in/kris-stern/[LinkedIn],
+link:https://contributors.jenkins.io/pages/contributors/kris-stern/[Contributor spotlight],
+link:https://community.jenkins.io/u/krisstern/summary[Jenkins community]
+
+[[oleg_nenashev]]
+=== link:/blog/authors/oleg_nenashev/[Oleg Nenashev]
```

Comment:
I will need to update the author profile a bit, too

Reply:
I believe the author profile update was done in:

* #7566

Reply:
Yep